frontend/src/metabase/parameters/utils/mbql.unit.spec.js

```diff
@@ -160,9 +160,11 @@ describe("parameters/utils/mbql", () => {
         "!=",
         ["field", null, { "temporal-unit": "month-of-year" }],
         date()
+          .date(1)
```

Comment:
I'm not quite familiar with MBQL yet but I guess `{ "temporal-unit": "month-of-year" }` only cares about the month part of `YYYY-MM-DD` so basically, the year and date is ignored correct?